README.md

```diff
@@ -75,13 +71,7 @@ A list of Java language features which are included in the analyzer can be found
 
 ## Support Libraries
 
-There is currently no support library available. 
-
-
-## Sample applications renderers
-This repository contains two sample applications that make use of the Java Spring framework ([Spring events](/ldj-spring-event-example) & [Spring AMQP](/ldj-spring-amqp-example)).
-For both these sample applications a [renderer](/ldj-spring-renderer-example) has been made (which generates a UML sequence diagram). This may serve as a source of inspiration for creating your own renderers.
-
```

Comment:
This seems like it is also still true and could be kept.

I would perhaps spell the header 'Sample applications **and** renderers'.